### PR TITLE
Refactored backend work

### PR DESCRIPTION
**Pull request by dbvora03**
Someone pls approve 😊

### CODE REVIEW

File: Python_Backend/data_collection/__init__.py [hunk 1]
```diff
@@ -13,29 +16,31 @@ def __init__(self, name):
...
-        db.autocommit = True
-        cursor = db.cursor()
-        cursor.execute(open("./db_schema/schema.sql", "r").read(),
-                       multi=True)  # Run Database Schema to Create it
-        cursor.close()  # Close
+        sensor_readings_collection = mydb['sensor_readings']
 
-    def upload_sensor_data(self, readings):
+    def upload_sensor_data(self, arduino_name, arduino_id, sensor_type, sensor_id, reading):
         query = "INSERT INTO sensor_readings (arduino_name,arduino_id,sensor_type,sensor_id,reading,time)" \
```
Comment (alvanli) [inline Python_Backend/data_collection/__init__.py:31]: delete unused query

File: Python_Backend/config.py [hunk 2]
```diff
@@ -28,17 +28,32 @@
...
 if use_i2c:
     arduinos = arduino_interface.arduino_interface
 else:
     arduinos = arduino_interface.fake_arduino_interface
 
+arduinos = arduino_interface.fake_arduino_interface
+
+logger = logging.get_logger("Database connection")
+try:
+    dbclient = pymongo.MongoClient('mongodb+srv://watlock:general@cluster0.7s0cr.mongodb.net/myFirstDatabase?retryWrites=true&w=majority')
```
Comment (alvanli) [inline Python_Backend/config.py:39]: this prob shdnt be out like this, put in env?

**Review by alvanli — COMMENTED** [review]
Change Readme to reflect changes + change requirements.txt in Tools

(havent tested yet)

File: Python_Backend/config.py (hunk 2 above)
Comment (dbvora03) [reply Python_Backend/config.py:39]: oops forgot to change that, ty

**Pull request merged**